CONTRIBUTING.md

```diff
@@ -73,6 +73,10 @@ There will be a notification in the slack channel `#dsp-callisto-internal` once
 
 The terra-docker image hash update [job](https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/)  mentioned above determines which images (including derived images) have been changed and builds each image with the appropriate tag.
 
+## Updating terra-docker-versions-*.json Best Practices
+
+As of 6/8/2022, we have added a new process for updating the image versions available across different environments. Before the Leo PR is merged into develop, the user will need to generate the `terra-docker-versions-new.json` with the new images and copy it by doing `gsutil cp terra-docker-image-documentation/terra-docker-versions-new.json terra-docker-image-documentation/terra-docker-versions-candidate.json`. The remainder of the update process will be covered through Leo deployment. As part of reliability changes, the versions file will live in the corresponding bucket with the following name convention: `terra-docker-image-documentation-{environment}`.
```

Comment:
https://github.com/DataBiosphere/terra-docker/pull/326#discussion_r891529079 and it should be `gsutil cp gs://..`